joomla/joomla-cms
# [4.0] Group toolbar buttons in com_plugins

### Summary of Changes
This PR continues the grouping of action buttons in the backend (see #22832, #22393, #23337 and #23336, #23374, #23368, #23380). This time the plugins overview.

### Testing Instructions
Go to the plugins overview in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select one or more plugins.
